app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for char in phrase.lower():
+        if char == letter.lower():
+            count += 1
+    return count
```

Comment:
yeah, this is working solution, but it's better to use .count() method of string. It'll simplify your code ;)